CHANGELOG.md

```diff
@@ -3,8 +3,10 @@
 ### unreleased ###
 * **English**
 	 * Option to provide a custom key for the Google Safe Browsing API
+	 * Scan files of parent theme if a child theme is active
 * **Deutsch**
 	* Möglichkeit einen eigenen Schlüssel für die Google Safe Browsing API zu verwenden
+	* Dateien des Parent Themes scannen, falls ein Child Theme aktiv ist
```

Comment:
Vielleicht 'Eltern' Theme, aber ich kenne die Standard-Übersetzungen von WP nicht wirklich gut.

Reply:
Ich kann in den WP Core Translations nur an einer Stelle “übergeordnetes Theme“ finden.
“Child Theme“ wird meist zu “Child-Theme“.

Reply:
Da habe ich keinen Plan, war mir nur im Englisch Deutsch aufgefallen, was man halt schwer weg bekommt. Einfach so nah an die vier Übersetzung wir möglich, da versucht man sich dich ganz gut.

Reply:
> Dateien des übergeordneten Themes scannen, falls ein Child-Theme aktiv ist

Damit ist es denke ich so wenig unnötige Sprachmischung wie nötig und nah genug am Core.